src/combine.py

```diff
@@ -37,6 +37,12 @@
 from datetime import timedelta # To store the parsed duration of files and calculate the accumulated duration
 from random import shuffle # To be able to shuffle the list of files if the user requests it
 import csv # To use for the cutpoint files they are CSV files
+import platform # identify which platform we're running on
+import tempfile # platform-independent tempoary files and directories
+
+# default filenames
+MP4BOX_DEFAULT_FILENAME = "mp4box" if not (platform.system() == "Windows") else "mp4box.exe"
```

Comment:
Consider simplifying the conditional here to not use a negative, easier to read, also consider making the comparison case insensitive, i.e. 
```
"mp4box.exe" if platform.system().casefold() == "Windows".casefold() else "mp4box"
```